README.md

```diff
@@ -792,7 +793,8 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [mpd](http://www.musicpd.org/) - Daemon to remotely play music, stream music, handle and organize playlists. Many clients available. `GPLv2` `C++`
   * [mStream](http://mstream.io/) - Music streaming server with GUI management tools. Runs on Mac, Windows, and Linux. ([Demo](https://darncoyotes.mstream.io/), [Source Code](https://github.com/IrosTheBeggar/mStream)) `GPLv2` `Nodejs`
   * [Music Browser](https://github.com/henrik242/musicbrowser) - Browser and streamer for your music collection. It is runs on most operating systems, and is light enough to run flawlessly on NAS devices. `GPLv3` `PHP`
-   * [ympd](http://www.ympd.org/) - Standalone MPD Web GUI written in C, utilizing Websockets and Bootstrap/JS. ([Source Code](https://github.com/notandy/ympd)) `GPLv2` `C`
+  * [musikcube](https://musikcube.com/) - Terminal-based music player, library, and streaming audio server. ([Source code](https://github.com/clangen/musikcube)) `BSD` `C++`
+  * [ympd](http://www.ympd.org/) - Standalone MPD Web GUI written in C, utilizing Websockets and Bootstrap/JS. ([Source Code](https://github.com/notandy/ympd)) `GPLv2` `C`
```

Comment:
Alrady listed at https://github.com/Kickball/awesome-selfhosted#audio-streaming

Reply:
I am not adding ympd anew, I am fixing a typo (an extra space character) in the line on which ympd was listed so that its indentation is perfectly consistent with the rest of the list.

Reply:
I've read the diff wrong. Thanks